ts/sagas/wallet.ts

```diff
@@ -284,6 +308,40 @@ function* showTransactionSummaryHandler(
   yield put(navigateTo(ROUTES.PAYMENT_TRANSACTION_SUMMARY));
 }
 
+function* showWalletOrSelectPsp(idWallet: number, paymentId?: string) {
+  const wallet: Option<Wallet> = yield select(specificWalletSelector(idWallet));
+  if (wallet.isSome()) {
+    // TODO: fetch list of PSPs available here
```

Comment:
story?

Reply:
Yeah I forgot to attach the link to the TODOs after a while, it's the same story for both cases ([this one](https://www.pivotaltracker.com/story/show/159494746)). Added it :)